packages/adapter-node/index.js

```diff
@@ -49,16 +53,20 @@ export default function (opts = {}) {
 
 			const pkg = JSON.parse(readFileSync('package.json', 'utf8'));
 
-			await esbuild.build({
-				platform: 'node',
-				sourcemap: 'linked',
-				target: 'es2022',
-				entryPoints: [`${tmp}/index.js`, `${tmp}/manifest.js`],
-				outdir: `${out}/server`,
-				splitting: true,
+			const bundle = await rollup({
```

Comment:
can we add a comment explaining why a second bundling step is needed

Reply:
done